framework/src/window.cpp

```diff
@@ -7,13 +7,13 @@ namespace gvk
 	void window::enable_resizing(bool aEnable)
 	{
 		mShallBeResizable = aEnable;
-		
+
 		if (is_alive()) {
-			mRecreationRequired = true;
+			glfwSetWindowAttrib(mHandle.value().mHandle, GLFW_RESIZABLE, aEnable ? GLFW_TRUE : GLFW_FALSE);
```

Comment:
That might not work if _Gears-Vk_ is running in multi-threaded mode. Please test it like follows:
In `composition.hpp`, disable single-threaded mode:
```
#define SINGLE_THREADED 0
```
and use `context_generic_glfw::dispatch_to_main_thread` to have `glfwSetWindowAttrib` be called from the main thread. (Sorry, this is not well documented.)

The point is that input and rendering run on different threads.

Reply:
Seems to still work.
I kept the `dispatch_to_main_thrread` in, but set gears back to single threaded in the commit.